packages/web-app-files/src/components/Spaces/SpaceHeader.vue

```diff
@@ -139,6 +143,7 @@ export default defineComponent({
     const { getFileContents, getFileInfo } = clientService.webdav
     const previewService = usePreviewService()
     const resourcesStore = useResourcesStore()
+    const { getDefaultAction } = useFileActions()
     const { actions: editReadmeContentActions } = useSpaceActionsEditReadmeContent()
```

Comment:
I guess this can be removed then.

I'm wondering in general, now that this is an `a` element, wouldn't it make sense in the context menu as well? Or is there a reason it should differ?

Reply:
Would make sense, was there and encounter issues. In the context menu action, we care about creating the .space folder and the readme file if not exist beforehand. 

As soon those files are protected from being deleted:
https://github.com/owncloud/ocis/issues/8719

We might be able to convert the button in the context menu as well to an a tag